modules/Bio/Otter/Auth/SSO.pm

```diff
@@ -130,7 +130,7 @@ Note that this rolls authentication and authorisation into one lump.
 =cut
 
 sub auth_user {
-    my ($called, $sangerweb, $Access) = @_;
+    my ($called, $Access, $unauthorized_user) = @_;
```

Comment:
'Access' is a regular variable, shouldn't have first capital letter

Reply:
I didn't create this variable. It already exists.